File: ProgramingBasics/SimpleConditionalStatements/04.GreaterNumber/src/GreaterNumber.java
import java.util.Scanner;

public class GreaterNumber {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter two integers:");
        int num1 = Integer.parseInt(scanner.nextLine());
        int num2 = Integer.parseInt(scanner.nextLine());

        if (num1 > num2) {
            System.out.println("Greater number: " + num1);
        }
        else {
            System.out.println("Greater number: " + num2);
        }
    }
}

File: ProgramingBasics/SimpleConditionalStatements/04.GreaterNumber/04.GreaterNumber/GreaterNumber.cs
using System;

namespace _04.GreaterNumber
{
    public class GreaterNumber
    {
        public static void Main(string[] args)
        {
            Console.WriteLine("Enter two integers:");
            var num1 = int.Parse(Console.ReadLine());
            var num2 = int.Parse(Console.ReadLine());

            if (num1 > num2)
            {
                Console.WriteLine($"Greater number: {num1}");
            }
            else
            {
                Console.WriteLine($"Greater number: {num2}");
            }
        }
    }
}
